src/assets/wise5/components/dialogGuidance/edit-dialog-guidance-computer-avatar/edit-dialog-guidance-computer-avatar.component.html

```diff
@@ -1,46 +1,44 @@
-<div fxLayout="column" fxLayoutGap="16px">
+<div class="edit-dialog-guidance" fxLayout="column" fxLayoutGap="16px">
   <mat-checkbox
     color="primary"
     [(ngModel)]="computerAvatarSettings.useGlobalComputerAvatar"
     (ngModelChange)="componentChanged()"
     i18n
     >Use Global Computer Avatar</mat-checkbox
   >
-  <mat-form-field class="avatar-label" appearance="fill">
-    <mat-label i18n>Computer Avatar Label</mat-label>
-    <input
-      matInput
-      [(ngModel)]="computerAvatarSettings.label"
-      (ngModelChange)="componentChanged()"
-      placeholder="Thought Buddy"
-      i18n-placeholder
-    />
-    <mat-hint i18n>This is what the computer avatar will be referred as</mat-hint>
-  </mat-form-field>
-  <mat-form-field appearance="fill">
-    <mat-label i18n>Computer Avatar Prompt</mat-label>
-    <textarea
-      matInput
-      [(ngModel)]="computerAvatarSettings.prompt"
-      (ngModelChange)="componentChanged()"
-      placeholder="Discuss your answer with a thought buddy!"
-      i18n-placeholder
-      cdkTextareaAutosize
-    >
-    </textarea>
-    <mat-hint i18n>This text will introduce the avatar selection task</mat-hint>
-  </mat-form-field>
-  <mat-form-field appearance="fill">
-    <mat-label i18n>Computer Avatar Initial Response</mat-label>
-    <input
-      matInput
-      [(ngModel)]="computerAvatarSettings.initialResponse"
-      (ngModelChange)="componentChanged()"
-      placeholder="Hi there! It's nice to meet you. What do you think about..."
-      i18n-placeholder
-    />
-    <mat-hint i18n>Computer avatar will initiate dialog with this text</mat-hint>
-  </mat-form-field>
+  <translatable-input
+    [content]="computerAvatarSettings"
+    key="label"
+    label="Computer Avatar Label"
+    i18n-label
+    placeholder="Thought Buddy"
+    i18n-placeholder
+    (defaultLanguageTextChanged)="componentChanged()"
+    class="name"
+    hint="This is what the computer avatar will be referred to as"
```

Comment:
For all the hints in this file, should we add a i18n-hint attribute alongside them?